i6_models/parts/rasr_fsa.py

```diff
@@ -42,6 +42,51 @@ def to(self, device: Union[str, torch.device]) -> WeightedFsa:
         return WeightedFsa._make(tensor.to(device) for tensor in self)
 
 
+class WeightedFsaV2(NamedTuple):
+    """
+    Convenience class that represents an FSA for the fbw2 loss from i6_native_ops. It supports scaling the weights of
+    the FSA by simple left-multiplication and moving the tensors to a different device.
+    It can be passed to :func:`i6_native_ops.fbw2.fbw2_loss`.
+
+    :param num_states: the number of states per sequence (shape: [N])
+    :param num_edges: the number of edges per sequence (shape: [N])
+    :param edges: a [3, E] tensor of edges with number of edges E and where each column is an edge
```

Comment:
The [docstring of fbw_loss2](https://github.com/rwth-i6/i6_native_ops/blob/d70308a0722f0632f70a690974f3c51444664e13/i6_native_ops/fbw2/__init__.py#L77) says that the expected shape is [4,E] + the sequence idx

Reply:
Ups, this is an error in the documentation of fbw_loss2. The new op does not have the sequence ID anymore for each edge separately. It's rather a separate tensor in the FSA.

Reply:
Ah yes that's what I remember from reviewing the code, as well, just wanted to make sure.